guidelines/raster.md

```diff
@@ -1,11 +1,37 @@
 # Contributing Raster Data
 
 ## Data structure
-Cloud Optimized GeoTIFF need to:
+All raster data for the site is stored as [Cloud Optimized GeoTIFF](https://www.cogeo.org/) (COG). One way to validate that data is in the proper format is using [rio-cogeo](https://github.com/cogeotiff/rio-cogeo):
 
-- pass validation by `rio cogeo`
-- have `nodata` values set
+- First, check that it passes validation with `rio cogeo validate my_raster.tif`
+- Then ensure that it has a `nodata` value set. You can see the nodata value like so:
+
+```sh
+rio insp my_raster.tif
+>>> print(src.meta)
+```
+
+This same library can also create a Cloud Optimized GeoTIFF with the following command:
+
+```sh
+rio cogeo create my_raster.tif my_cog_raster.tif
+```
 
 ## Naming convention
 
+New raster files are added to the dashboard manually so the naming convention is rather liberal. The only requirement is that for date-specific data, the file name must include the date, formatted as:
+- `YYYYMM` for monthly data
+- `YYYY_MM_DD` for sub-monthly data (anything with daily or semi-periodic data)
+
+If the file doesn't have global coverage, please use a portion of the file name to indicate the spotlight area it covers. We provide data for the following [spotlight areas](https://8ib71h0627.execute-api.us-east-1.amazonaws.com/v1/sites). For example:
+
+```sh
+my_tif_tk_2020_02_19.tif # Data for Tokyo on February 19th, 2020.
+```
+
 ## Delivery mechanism
+
+There are three mechanisms for making raster data available through this API:
+- **publicly available**: any publicly available COG can be accessed through this COG. Because the API is run on Amazon Web Services in the `us-east-1` region, data hosted on S3 in this region will have faster response times to the API.
```

Comment:
```suggestion
- **publicly available**: any publicly available COG can be accessed through this API. Because the API is run on Amazon Web Services in the `us-east-1` region, data hosted on S3 in this region will have faster response times to the API.
```